google/cloud/spanner/BUILD

```diff
@@ -64,6 +64,14 @@ load(":spanner_client_unit_tests.bzl", "spanner_client_unit_tests")
     ],
 ) for test in spanner_client_unit_tests]
 
+cc_binary(
```

Comment:
This binary probably belongs in `google/cloud/spanner/examples` or `examples/`, consider moving it or add a TODO to move it.